src/components/cards/CommentCard.tsx

```diff
@@ -0,0 +1,245 @@
+import React from "react";
+import { FontCSS, TdCSS } from "../../css";
+import { palette } from "@guardian/src-foundations";
+import { Content, Tag } from "../../api";
+import { formatImage } from "../../image";
+import sanitizeHtml from "sanitize-html";
+import { Table, RowCell, TableRowCell, TableRow } from "../../layout/Table";
+import { Headline } from "../../components/Headline";
+import { Image } from "../../components/Image";
+import { headline } from "../../styles/typography";
+
+type Size = "small" | "large";
+
+const tdStyle: TdCSS = {
+    backgroundColor: palette.neutral[100],
+    borderTop: `1px solid ${palette.opinion.main}`,
+    padding: "0"
+};
+
+const metaWrapperStyle = (size: Size): TdCSS => {
+    const rightPad = size === "large" ? "40px" : "10px";
+    const bottomPad = size === "large" ? "0" : "15px";
+    return {
+        padding: `3px ${rightPad} ${bottomPad} 10px`
+    };
+};
+
+const standfirstStyle: TdCSS = {
+    padding: "20px 10px 10px 10px",
+    verticalAlign: "bottom"
+};
+
+const linkStyle: FontCSS = {
+    textDecoration: "none"
+};
+
+const spanStyle: FontCSS = {
+    ...headline({ level: 1 }),
+    color: palette.neutral[7]
+};
+
+const columnStyleRight: TdCSS = {
+    width: "30%",
+    verticalAlign: "bottom",
+    padding: "0"
+};
+
+interface Props {
+    content: Content;
+    salt: string;
+    size: "large" | "small";
+    shouldShowImage?: boolean;
+    shouldShowProfileImage?: boolean;
+}
+
+const brazeParameter = "?##braze_utm##";
+
+const TrailText: React.FC<{
+    text: string;
+    linkURL: string;
+    size: Size;
+}> = ({ text, linkURL }) => {
+    return (
+        <td className="m-pad" style={standfirstStyle}>
+            <a style={linkStyle} href={linkURL}>
+                {" "}
+                <span style={spanStyle}>{text}</span>
+            </a>
+        </td>
+    );
+};
+
+// TODO add alt text
+const ContributorImage: React.FC<{
+    src: string;
+    salt: string;
+    width: number;
+    alt: string;
+}> = ({ src, salt, width, alt }) => {
+    if (!src) {
+        return null;
+    }
+
+    const formattedImage = formatImage(src, salt, width);
+    return (
+        <Image
+            src={formattedImage}
+            width={width}
+            alt={alt}
+            pillar="Opinion"
```

Comment:
I don't think there's a good reason for this card to have anything hardcoded to the Opinion pillar. We have access to the pillar here, so we could pass it around as prop so we ensure any borders will render the appropriate pillar colour. This makes the card much more reusable - even if we don't have a use case for it outside of Opinion yet.